etc/config-template.toml

```diff
@@ -245,8 +245,12 @@
 # set backup path, if not set, use "backup" under store path.
 # backup-dir = "/tmp/tikv/store/backup"
 
+# Allows OS to incrementally sync files to disk while they are being
+# written, asynchronously, in the background.
+# bytes-per-sync = "1MB"
+
 # Allows OS to incrementally sync WAL to disk while it is being written.
-# wal-bytes-per-sync = 0
+# wal-bytes-per-sync = "32KB"
```

Comment:
64KB may be better in our benchmark?

Reply:
I checked the doc, 32KB is better.

Reply:
Depends on my benching result in raftengine, 512KB can achieve good performance, also can reduce IO operations per minutes. 512KB is a recommended value, but you also need benching result base on rocksdb. When you are benching, please don't just focus on QPS, you also need to pay close attentions to IOPS and utilization of disk.